WireDomain/Tests/WireDomainTests/Event Processing/FeatureConfigEventProcessor/FeatureConfigUpdateEventProcessorTests.swift

```diff
@@ -0,0 +1,110 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireAPI
+import WireAPISupport
+import WireDataModel
+import WireDataModelSupport
+import XCTest
+
+@testable import WireDomain
+
+final class FeatureConfigUpdateEventProcessorTests: XCTestCase {
+
+    var sut: FeatureConfigUpdateEventProcessor!
+
+    var coreDataStack: CoreDataStack!
+    let coreDataStackHelper = CoreDataStackHelper()
+    let modelHelper = ModelHelper()
```

Comment:
Conisder creating/destroying these in setup and teardown as I mention [here](https://github.com/wireapp/wire-ios/pull/1941#discussion_r1766846769).

Reply:
done thx